protostar-rust/src/main.rs

```diff
@@ -40,11 +40,12 @@ fn main_execution() -> Result<()> {
         .context("Failed to convert corelib path to Utf8PathBuf")?;
 
     let scarb_metadata = MetadataCommand::new().inherit_stderr().exec()?;
-
-    let _ = Command::new("scarb")
-        .current_dir(std::env::current_dir().expect("failed to obtain current dir"))
-        .arg("build")
-        .output()?;
+    // TODO: uncomment this, now there is mocked data built with scarb 0.5.0-alpha.1 (it supports cairo 2)
+    // but once the scarb is fully released, this should be uncommented and working properly
+    // let _ = Command::new("scarb")
+    //     .current_dir(std::env::current_dir().expect("failed to obtain current dir"))
+    //     .arg("build")
+    //     .output().expect("failed to execute process");
```

Comment:
Cannot we use unstable scarb by default? mocked data seems off

Reply:
Yes, especially since it's been ignored. Well, `scarb` here comes from the system, I wanted to explicitly point out that you have to have a certain scarb version installed in order to make it work.

Reply:
I uncommented it in [uncomment scarb command](https://github.com/software-mansion/protostar/pull/2103/commits/459a503c1919ee076363ed4c91a7f0892c8806fa), I assume that people seeing the error will know that they have to use different version of scarb, the error goes:
```
{"type":"error","message":"Version solving failed:\n- declare_test v0.1.0 (/private/var/folders/by/rbtvj__j5yxcm_cd6w91xj5w0000gn/T/.tmplwxGXw/Scarb.toml) cannot use starknet v1.1.0 (std), because declare_test requires starknet ^2.0.0-rc3\n\nScarb does not have real version solving algorithm yet.\nPerhaps in the future this conflict could be resolved, but currently,\nplease upgrade your dependencies to use latest versions of their dependencies."}
```